scripts/uberenv/uberenv.py

```diff
@@ -272,8 +271,6 @@ def detect_platform(self):
         res = None
         if is_darwin():
             res = "darwin"
-        elif is_wsl():
```

Comment:
This is the only case where it is used. We could have wsl users manually specify the spack config (this is what I do for WSL2 anyway), but we should update the docs to reflect this.

Reply:
yea we should just have them specify the directory in the docs.

Reply:
👍 Another PR is incoming to fix the docs.